spec/spec_helper.rb

```diff
@@ -1,4 +1,3 @@
-# frozen_string_literal: true
 require 'chefspec'
 require 'chefspec/berkshelf'
```

Comment:
Since you have this major update should we switch this cookbook to policyfile vs berkshelf? I think that'd polish this off nicely and make it a great example for community on how to write a modern cookbook, resources, libraries, etc...

Reply:
Personally, I'm not a massive fan of policyfile for library cookbooks (or policyfile full stop as an aside). I'm not sure what it'd give the cookbook that it doesn't get from berks?

Reply:
Other than future proofing it really doesn’t give anything over berks especially for library cookbooks.

Reply:
I think until it breaks we should leave all the cookbooks on Berks. See my haproxy PR. There was little benefit. 
But if we do it should be it's own pr

Reply:
I'm fine with that, just brought it up more for discussion.